src/app/settings/views/Repositories/RepositoriesList/RepositoriesList.test.tsx

```diff
@@ -133,20 +148,29 @@ describe("RepositoriesList", () => {
       </Provider>
     );
     // Click on the delete button:
-    wrapper.find("TableRow").at(3).find("Button").at(1).simulate("click");
+    await userEvent.click(
+      within(
+        screen.getByRole("row", {
+          name: "secret_archive http://www.website.com No Edit Delete",
+        })
+      ).getByTestId("table-actions-delete")
```

Comment:
This might be better to be `getByRole("button", {name: Label.Delete})` (which will require adding a Label enum in TableActions.tsx).

Reply:
The reason I didn't do that for this case is that there are two delete buttons for every row, both with the name "Delete", so getting them by role this way returns two of them. I could do getAllByRole and get them by list index but that feels a little janky to me

Reply:
Ah OK. One thing you could try is to add an `aira-label` to [each column that match the headers](https://github.com/canonical/maas-ui/blob/main/src/app/subnets/components/ReservedRanges/ReservedRanges.tsx#L124)  which you could then use to target the actions column: `.getAllByRole("gridcell", { name: Label.Actions })` and then find the delete button inside the column.

Reply:
Got this sorted!